composer.json

```diff
@@ -193,7 +195,7 @@
     "conflict": {
         "drupal/drupal": "*"
     },
-    "minimum-stability": "stable",
+    "minimum-stability": "dev",
```

Comment:
Is this necessary?

Reply:
Yes; config_split won't install otherwise (config_filter@rc depend). If I revert it after adding it, we'll get an outdated lock file error. Annoy, but there ya go.